code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -60,7 +67,10 @@ true, and the mob is not yet deleted, so we need to check that as well*/
 
 // Knifing
 /mob/living/carbon/proc/attack_throat(obj/item/W, obj/item/weapon/grab/G, mob/user)
-
+	//presumably, if they are wearing a helmet that stops pressure effects, then it probably covers the throat as well
```

Comment:
I think it's better be done with some new flag, checking for var that relates to atmos is generally bad idea.